packages/react/src/combobox/root/ComboboxRoot.test.tsx

```diff
@@ -1031,6 +1031,32 @@ describe('<Combobox.Root />', () => {
     });
   });
 
+  describe('prop: required', () => {
+    it('does not mark the hidden input as required when selection exists in multiple mode', async () => {
+      const { container } = await render(
+        <Combobox.Root multiple required name="languages" value={['a']}>
+          <Combobox.Input />
+        </Combobox.Root>,
+      );
+
+      const hiddenInput = container.querySelector<HTMLInputElement>('input[aria-hidden="true"]');
```

Comment:
Nitpick: Have you tried `screen.getByRole('textbox', { hidden: true })` instead? 🤔